CMakeLists.txt

```diff
@@ -512,88 +532,18 @@ endif()
 # RDC SUPPORT FOR VECGEOM+CUDA
 #----------------------------------------------------------------------------#
 
-# Load RDC utils using CMAKE_MODULE_PATH
-include(CudaRdcUtils OPTIONAL)
-
-# Load our local copy if it's newer
-if(NOT CUDA_RDC_VERSION)
-   set(_loaded_CUDA_RDC_VERSION 0)
-else()
-   set(_loaded_CUDA_RDC_VERSION ${CUDA_RDC_VERSION})
-endif()
```

Comment:
This was made 'unnecessary' because the optionality of the install of the `CudaRdcUtils` was inadvertently (!?) removed in https://github.com/celeritas-project/celeritas/pull/1674 and in particular [`f6fc29f` (#1674)](https://github.com/celeritas-project/celeritas/pull/1674/commits/f6fc29fa99c2d6bbe465d9ef5f2050f4a8aacc9c).  

That commit seems to address the issue "Always install RdcUtils since the build fails without it " but it is not clear that this was the best solution ... and the very least the commit log could use an extension to explain why installing  an older/obsolete version is useful.

Reply:
It was an expedient to fix the build. I'll update the PR notes. I'd like to get this in to enable `celeritas_add_library` downstream in our examples and tag an `rc`; then could you create a new PR that finesses the installation and reverts this chunk and https://github.com/celeritas-project/celeritas/pull/1674/commits/f6fc29fa99c2d6bbe465d9ef5f2050f4a8aacc9c if you can get everything working without it?

Reply:
Since we agreed to always distribute the local version, there is no longer a need to revert.  But whether we use below:
```
include(CudaRdcUtils OPTIONAL)
include("${_LOCAL_RDCUTILS_FILENAME}")
```
or just
```
include("${_LOCAL_RDCUTILS_FILENAME}")
```
depends on whether we choose to trust the `find_package` to already include their own version (i.e. in case theirs is newer).

Reply:
We haven't release any versions of VecGeom nor G4VG yet that include RdcUtils in their Config file. Future VG and G4VG should distribute and include it (and their inclusion will be new enough not to override extant newer versions). Therefore in the future a `find_package` alone will be enough to load the newest RdcUtils among all the dependencies. Present `find_package` is unnecessary because we know no present version is newer than ours.

Does that sound right?